rosdep/base.yaml

```
@@ -552,13 +552,15 @@ clang-format:
   alpine: [clang]
   arch: [clang]
   debian: [clang-format]
-  fedora: [clang]
+  fedora: [clang-tools-extra]
```

If we look at what binaries the `clang-format` tools provides on Ubuntu, it looks like it provides all of:

```
/usr/bin/clang-format
/usr/bin/clang-format-diff
/usr/bin/git-clang-format
```

On RHEL and Fedora, `clang-tools-extra` provides both `clang-format` and `clang-format-diff`, while `git-clang-format` comes in via the `git-clang-format` package.

Thus it seems to me that this should be:

```suggestion
  fedora: [clang-tools-extra, git-clang-format]
```

(the same goes for the `rhel` key below)

---

makes sense, done